DNN Platform/Library/UI/WebControls/PropertyEditor/Edit Controls/DNN Edit Controls/DNNLocaleEditControl.cs

```diff
@@ -190,9 +191,35 @@ protected override void RenderEditMode(HtmlTextWriter writer)
 			writer.AddAttribute(HtmlTextWriterAttribute.Class, "dnnLeft");
 			writer.RenderBeginTag(HtmlTextWriterTag.Div);
 
-			//Render the Select Tag
-			writer.AddAttribute(HtmlTextWriterAttribute.Name, UniqueID);
+		    IList<CultureInfo> cultures = new List<CultureInfo>();
+		    switch (ListType)
+		    {
+		        case LanguagesListType.All:
+		            var culturesArray = CultureInfo.GetCultures(CultureTypes.SpecificCultures);
+		            Array.Sort(culturesArray, new CultureInfoComparer(DisplayMode));
+		            cultures = culturesArray.ToList();
+                    break;
+		        case LanguagesListType.Supported:
+		            cultures = LocaleController.Instance.GetLocales(Null.NullInteger).Values
+		                .Select(c => CultureInfo.GetCultureInfo(c.Code))
+		                .ToList();
+		            break;
+		        case LanguagesListType.Enabled:
+		            cultures = LocaleController.Instance.GetLocales(PortalSettings.PortalId).Values
+		                .Select(c => CultureInfo.GetCultureInfo(c.Code))
+		                .ToList();
+		            break;
+		    }
+
+		    var promptValue = StringValue == Null.NullString && cultures.Count > 1 && !Required;
+
+            //Render the Select Tag
+            writer.AddAttribute(HtmlTextWriterAttribute.Name, UniqueID);
```

Comment:
Please take care to keep the formatting consistent. It confuses the diff when you change indentation and makes code harder to read